File: w8/p1.cpp
#include "Pythia8/Pythia.h"
#include <iostream>

using namespace Pythia8;
using namespace std;

int main(int argc, char* argv[]) {
    // Initialize Pythia
    Pythia pythia;
    // proton beam
    pythia.readString("Beams:idA = 2212");
    // proton beam
    pythia.readString("Beams:idB = 2212"); 
    // 13 TeV collisions
    pythia.readString("Beams:eCM = 13000."); 
    // higgs production -> on
    pythia.readString("HiggsSM:all = on"); 
    pythia.init();
    
    // h boson width,PDG ID 25 for h boson
    double higgsWidth = pythia.particleData.mWidth(25); 
    
    
    cout << "Width at 125 GeV = " << higgsWidth << " GeV" << endl;
    
    return 0;
}

File: w8/p1.py
import pythia8

pythia = pythia8.Pythia()
# default collison energy: 14 TeV
# enable hb production, gluon -> hb
pythia.readString("HiggsSM:all = on")
# taking only sm hb
pythia.readString("Higgs:useBSM = off")  
pythia.init()

# PDG ID for hb = 25
higgs_data = pythia.particleData.mWidth(25)

print(f"SM Higgs boson width for m_H = 125 GeV is {higgs_data:.6f} GeV")
# 0.004083428388965407 GeV